exasol_advanced_analytics_framework/udf_communication/peer_communicator/background_listener_thread.py

```diff
@@ -112,16 +116,16 @@ def run(self):
         try:
             self._run_message_loop()
         finally:
-            self._close()
+            self._stop()
 
-    def _close(self):
+    def _stop(self):
         self._logger.info("start")
```

Comment:
start?

Reply:
the "start" is the event of this logger call, the logging adds module and function name automatically